core/src/main/java/hudson/model/FingerprintCleanupThread.java

```diff
@@ -79,7 +79,7 @@ public void execute(TaskListener listener) {
 
         if (!(FingerprintStorage.get() instanceof FileFingerprintStorage) &&
                 FingerprintStorage.getFileFingerprintStorage().isReady()) {
-            FileFingerprintStorage.getFileFingerprintStorage().iterateAndCleanupFingerprints(listener);
+            FingerprintStorage.getFileFingerprintStorage().iterateAndCleanupFingerprints(listener);
```

Comment:
This PR already provides benefits: Revealing an obvious bad name in the API.

https://github.com/jenkinsci/jenkins/blob/6f2e7b0085e39d02a92312a57a5cf7f0b84dfc05/core/src/main/java/jenkins/fingerprints/FingerprintStorage.java#L61-L63